lib/inertia_rails/renderer.rb

```diff
@@ -45,7 +45,7 @@ def layout
     def computed_props
       _props = ::InertiaRails.shared_data(@controller).send(prop_merge_method, @props).select do |key, prop|
         if rendering_partial_component?
-          key.in? partial_keys
+          key.to_sym.in? partial_keys
```

Comment:
`partial_keys` are cast to symbols. In #99 , we changed `@props` to be a hash with indifferent access. That casts the keys of `@props` to strings. Thus, `computed_props` was filtering every single prop out, completely breaking partial reloads.